galsim/wcs.py

```diff
@@ -214,30 +214,38 @@ def toWorld(self, arg, **kwargs):
         else:
             return self.posToWorld(arg, **kwargs)
 
-    def posToWorld(self, image_pos, **kwargs):
+    def posToWorld(self, image_pos, color=None, **kwargs):
```

Comment:
Right, no.  The type of the color is defined by the WCS type that uses it.  Gary's WCS's use a scalar float.  But a full set of several colors would be fine.  Or even some user-defined class if necessary.  I think I don't limit the implementation of that to anything specific.